crypto/s2n_ecdsa.c

```diff
@@ -32,8 +35,29 @@
 #include "crypto/s2n_openssl.h"
 #include "crypto/s2n_pkey.h"
 
+
 #define S2N_ECDSA_TYPE 0
 
+S2N_RESULT s2n_unsafe_ecdsa_get_mut(const struct s2n_ecdsa_key *ecdsa_key, EC_KEY **out_ec_key) {
```

Comment:
You're going to need a "why" comment explaining the old / new openssl behavior, probably on this function. We don't want future us to have to puzzle through why this is necessary again.

Reply:
https://github.com/aws/s2n-tls/pull/3474/commits/0070d164f9dfa7eb7f22251abec6e87f63d663ad